client/viewer/userconfig.py

```diff
@@ -71,7 +71,7 @@ def index_core(request):
     item_name_list = request.GET.getlist('items[]')
 
     body = json.dumps(UserConfig.get_items(item_name_list, user_id))
-    return HttpResponse(body, mimetype='application/json')
+    return HttpResponse(body, content_type = 'application/json')
```

Comment:
PEP8, a coding style for python code this project also uses, suggests not to insert spaces around '='.
> Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. 

https://www.python.org/dev/peps/pep-0008/